download-data.sh

```diff
@@ -1,19 +1,35 @@
 #!/usr/bin/env bash
 
-unameOut="$(uname -s)"
+OS_NAME="$(uname -s)"
 
-case "${unameOut}" in
-    Linux*)     sudo apt-get update && sudo apt-get install curl;;
-    CYGWIN*)    sudo apt-get update && sudo apt-get install curl;;
-    MINGW*)     sudo apt-get update && sudo apt-get install curl;;
-esac
+if [[ "$OS_NAME" == "Linux" ]]; then
+    sudo apt-get update && sudo apt-get install curl
```

Comment:
hmm. ok, but i feel like it's easier to just assume curl is installed

Reply:
It's not per default on Linux, that's why I added it.